change/@ni-nimble-components-6bd07a0a-7f59-4440-989a-516fca7124bf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Create WaferMap component",
+  "packageName": "@ni/nimble-components",
+  "email": "eliza.olariu@ni.com",
```

Comment:
In [the root CONTRIBUTING.md](https://github.com/ni/nimble/blob/main/CONTRIBUTING.md#beachball-change-file) there are instructions about how to configure GitHub to use a private email address to avoid exposing your NI address in public. For this PR you can manually modify this JSON file to copy in the address that GitHub shows at https://github.com/settings/emails

Reply:
Updated